healthworkforce/features/utils.js

```diff
@@ -0,0 +1,151 @@
+'use strict'
+
+const axios = require('axios')
+
+const OPENHIM_PROTOCOL = process.env.OPENHIM_PROTOCOL || 'http'
+const OPENHIM_API_HOSTNAME = process.env.OPENHIM_API_HOSTNAME || 'localhost'
+const OPENHIM_TRANSACTION_API_PORT =
+  process.env.OPENHIM_TRANSACTION_API_PORT || '5001'
+const CUSTOM_TOKEN_ID = process.env.CUSTOM_TOKEN_ID || 'test'
+
+const practitionerId = 'P10004'
+const practitionerRoleId = 'PR10001'
+const locationId = '2'
+const organizationId = '3'
+
+exports.triggerSync = async () => {
+  await axios({
+    url: `${OPENHIM_PROTOCOL}://${OPENHIM_API_HOSTNAME}:${OPENHIM_TRANSACTION_API_PORT}/mcsd-trigger`,
+    method: 'GET',
+    headers: {
+      'Content-Type': 'application/json'
+    }
+  })
+
+  // Allow the syncing to finish. The data will only be available in FHIR after a few seconds
+  await new Promise ((resolve, _reject) => {
+    setTimeout(() => resolve(), 60000)
+  })
+}
+
+exports.ihrisMockServicePractitioner = async () => {
+  // Mock ihris service running in docker container
```

Comment:
For this step definition we still need the code that creates the FHIR resource - we should POST a FHIR resource that the mock service is already returning (or modify one that it's returning to match). The mock service can just accept the create request and do nothing with it. When we finally do have actual services, then all should just work when we remove the mock services.

Reply:
addressed